server/testsupport/educationResponseTestDataBuilder.ts

```diff
@@ -0,0 +1,13 @@
+import type { EducationResponse } from 'educationAndWorkPlanApiClient'
+import EducationLevelValue from '../enums/educationLevelValue'
+import QualificationLevelValue from '../enums/qualificationLevelValue'
+
+const aValidEducationResponse = (): EducationResponse => ({
+  reference: 'dea24acc-fde5-4ead-a9eb-e1757de2542c',
+  educationLevel: EducationLevelValue.SECONDARY_SCHOOL_TOOK_EXAMS,
+  qualifications: [{ subject: 'Pottery', grade: 'C', level: QualificationLevelValue.LEVEL_4 }],
+  updatedAt: new Date('2023-06-19T00:00:00Z').toISOString(),
+  updatedByDisplayName: 'Alex Smith',
+})
```

Comment:
I think this might be missing a few fields 🤔 
According to our swagger spec (and hopefully the `EducationResponse` type that is generated from the swagger spec), it should look a bit more like this:
```
{
  "createdBy": "asmith_gen",
  "createdByDisplayName": "Alex Smith",
  "createdAt": "2023-06-19T09:39:44Z",
  "createdAtPrison": "BXI",
  "updatedBy": "asmith_gen",
  "updatedByDisplayName": "Alex Smith",
  "updatedAt": "2023-06-19T09:39:44Z",
  "updatedAtPrison": "BXI",
  "reference": "814ade0a-a3b2-46a3-862f-79211ba13f7b",
  "educationLevel": "PRIMARY_SCHOOL",
  "qualifications": [
    {
      "reference": "814ade0a-a3b2-46a3-862f-79211ba13f7b",
      "subject": "Maths GCSE",
      "level": "ENTRY_LEVEL",
      "grade": "Distinction",
      "createdBy": "asmith_gen",
      "createdAt": "2023-06-19T09:39:44Z",
      "updatedBy": "asmith_gen",
      "updatedAt": "2023-06-19T09:39:44Z"
    }
  ]
}
```